frontend-zs/src/components/stories/StoryList.component.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { Button, Container, Row, Col, Card, Media, Image, Form } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+import likeIcon from '../../icon_like.png'
+import './Story.scss'
+
+const StoryCard = props => (
+    <Col sm={6}>
+        <Card className="mb-4">
+            <Card.Header style={{ backgroundColor: '#5F696A' }} />
+            <Card.Body>
+                <Card.Title><strong>{props.story.title}</strong></Card.Title>
+                <Card.Subtitle className="mb-2 text-muted">
+                    <strong>{props.story.author}</strong>
+                </Card.Subtitle>
+                <Card.Text>{props.story.shortDescription}</Card.Text>
+                <Container>
+                    <Row className="justify-content-between">
+                        <Col>
+                            <Button variant="flat" size="sm" active>
+                                <Link to={"/geschichte/" + props.story._id}>
+                                    Geschichte lesen
+                                </Link>
+                            </Button>
+                        </Col>
+                        <Media>
+                            <Image
+                                className="mr-1"
+                                src={likeIcon}
+                                width="23"
+                                height="23"
+                                fluid
+                            />
+                            <Media.Body>
+                                <h5><strong>{props.story.numberOfLikes.length}</strong></h5>
+                            </Media.Body>
+                        </Media>
+                    </Row>
+                </Container>
+            </Card.Body>
+        </Card>
+    </Col>
+)
+
+export default class StoryList extends React.Component {
```

Comment:
I think we should be using functional components instead of class components, so that it uses less lines of code  which makes it easier to read..

Reply:
I am not that familiar with functional components and I have seen that the other group also used class components.

Reply:
I think we should discuss this with them then

Reply:
Having a decision on this seems important before merging the code. Code consistency is a major point of focus when the two teams are working independently.